pkgs/test/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.6.12
```

Comment:
[nit] We'll want to go with 1.7.0 since this is technically a new feature.

Reply:
👍 Done